root/programs/VB/Samples/WebApp_sample/ProjectX_sample/App_Code/sample/Dao/LayerD.vb

```diff
@@ -118,15 +118,15 @@ Public Class LayerD
     ''' <summary>件数情報を返すSELECTクエリを実行する</summary>
     ''' <param name="testParameter">引数クラス</param>
     ''' <param name="testReturn">戻り値クラス</param>
-    Public Sub SelectCount(ByVal testParameter As TestParameterValue, ByVal testReturn As TestReturnValue)
+    Public Sub SelectCount(testParameter As TestParameterValue, testReturn As TestReturnValue)
         ' ↓DBアクセス-----------------------------------------------------
 
         Dim filename As String = ""
 
-        If (testParameter.ActionType.Split("%"c))(2) = "static" Then
+        If (testParameter.ActionType.Split("%"C))(2) = "static" Then
```

Comment:
@yusukemaegawa 

This purpose was to eliminate the difference.
But, this did not be adjusted to "default side".

But, this changes passed the build process.
And, there is no problem on behavior.

Therefore, this time, I request you to merge this pullrequest in this state.